src/clusterfuzz/_internal/platforms/android/fetch_artifact.py

```diff
@@ -163,13 +168,34 @@ def get_client():
 
   return client
 
+def get_stable_build_info():
+  """Return stable artifact for cuttlefish branch and target."""
+  stable_build_info = STABLE_CUTTLEFISH_BUILD
+  gcs_build_info_url = DEFAULT_STABLE_CUTTLEFISH_BUILD_INFO
+
+  gcs_url_override = (
+    environment.get_value('OVERRIDE_STABLE_CUTTLEFISH_BUILD_INFO')
+    )
+  if gcs_url_override:
+    gcs_build_info_url = gcs_url_override
+
+  build_info_data = storage.read_data(gcs_build_info_url)
+  if build_info_data:
+    stable_build_info = json.loads(build_info_data)
```

Comment:
Again, there is a chance that the data is not formatted correctly in json. Should we consider wrapping this in a try / catch block as well?